javascript/builtins/AsyncFunction.json

```diff
@@ -0,0 +1,114 @@
+{
+  "javascript": {
+    "builtins": {
+      "AsyncFunction": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/AsyncFunction",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
```

Comment:
webview_android is also 55.

https://www.chromestatus.com/features/5643236399906816